Makefile

```diff
@@ -59,3 +66,73 @@ gscholar_alerts/extracted_citations.jsonl: gscholar_alerts/eml/
 
 gscholar_alerts/citations.jsonl: gscholar_alerts/extracted_citations.jsonl
 	jq -c 'select(.title != null and .authors != null) | del(.idx, .date, .data, .ref, .link)' $< >$@
+
+
+
+# HF Dataset Repos Updating
+
+# Since we dont know which years will exist in the jsonl,
+# better to use a sentinel file to track their timestamp as well.
+extract-citations.done: gscholar_alerts/citations.jsonl
+	mkdir -p $(TRACKED_FILES_BASEDIR)/citations
+	BASEDIR=$(TRACKED_FILES_BASEDIR)/citations ; \
+	YEARS=$$(cat tmp/citations.jsonl | jq -r ."year" | sort | uniq) ; \
+	for YEAR in $$YEARS; do \
+		jq -c "select(."year" == \"$$YEAR\")" tmp/citations.jsonl > "$$BASEDIR/$$YEAR.jsonl"; \
+	done
+	touch $@
+
+# TODO: Slightly inefficient now, if one .bib file has newer timestamp, redoes every year's .csv file.
+#       However we know that git (HF) updates only files that really changed, so the inefficiency is only in this step.
+#       Change?
+# TODO: Do NOT push file 2024 for now, we get an error in export-csv.py for this, resulting in an empy file.
+#       Correct this after bibtex code is corrected.
+extract-citations-annotated.done: $(BIBSRC)
+	mkdir -p $(TRACKED_FILES_BASEDIR)/citations-annotated
+	BASEDIR=$(TRACKED_FILES_BASEDIR)/citations-annotated ; \
+	for BIBFILE in $(BIBSRC); do \
+		if [ `basename $$BIBFILE .bib` = "cc2024" ]; then \
```

Comment:
FYI: the BibTeX format error should now be fixed. The PR branch needs to be rebased onto the recent "main" branch.